src/tutorials/data/queries.md

```diff
@@ -0,0 +1,365 @@
+
+In the following, we assume a DocType named `io.cozy.todos`  describing Todo lists.
+
+Here is a basic example of a cozy-client query returning all the Todo lists in a Cozy:
+```javascript
+const queryDef = client.find("io.cozy.todos")
+const docs = await client.queryAll(queryDef)
+```
+
+In this example, the `QueryDefinition` defines the query to get all the documents stored in the Todos database, expressed through `find("io.cozy.todos")`.
+
+
+## Selectors
+
+To use the Mango query system, you define a `selector`, that allows to express criterias to filter the documents to return.
+
+### Selectors with cozy-client
+
+With cozy-client, a selector is expressed through the [`where`](https://docs.cozy.io/en/cozy-client/api/cozy-client/#querydefinitionwhereselector-querydefinition) method, that takes into input an object with the selector expression. For instance, this query means “the todos in 'sport’ category AND a title ‘Exercices’”: 
+
+```javascript
+const queryDef = client
+  .find("io.cozy.todos")
+  .where({
+    "category": "sport",
+    "title": "Exercices"
+  })
+```
+
+`find` targets the Todos database, while `where` filters the documents to return.
+
+ℹ️ When running this query, an index will automatically be created on `category` and `title` fields. See the [index section](#index-fields) for more details about it. As mentioned in the [performances](#mango-performances), it is highly recommended to read it in order to avoid designing poorly efficient queries.
+
+ℹ️ The `$eq` and `$and` operators are implicit by default, which allow to write less verbose queries. Here is the explicit version of the previous query:
+
+```javascript
+const queryDef = client
+  .find("io.cozy.todos")
+  .where({
+    "$and": [
+      {
+        "category": {
+          "$eq": "sport"
+      },
+      {
+        "title": {
+          "$eq": "Exercices"
+        }
+      }
+    ]
+  })
+```
+
+### Mango operators
+
+There are two kinds of mango operators:
+
+- [Condition operators](https://docs.couchdb.org/en/2.3.1/api/database/find.html#condition-operators) (`$eq`, `$gt`, `$lt`, …)
+- [Combinaison operators](https://docs.couchdb.org/en/2.3.1/api/database/find.html#combination-operators) (`$and` , `$or`, `$not`, …)
+
+For more details and examples of selectors, you can directly reach the [CouchDB documentation](https://docs.couchdb.org/en/2.3.1/api/database/find.html#selector-syntax).
+
+### Mango performances
+
+Depending on your queries and their complexity, performances can be dramatically impacted, by serveral orders of magnitude, especially for large databases, i.e. starting for thousands of documents.
+Thus, it is highly recommended to take some time to understand how to [index fields](#index-fields) and the related [performances](./advanced.md#indexes-performances-and-design).
```

Comment:
```suggestion
It is highly recommended to take some time to understand how to [index fields](#index-fields) and the related [performances](./advanced.md#indexes-performances-and-design).
```

"Thus" is a bit formal.

Reply:
I don't really see the problem with being formal :)